pkgs/applications/misc/archiver/default.nix

```diff
@@ -5,16 +5,18 @@
 
 buildGoModule rec {
   pname = "archiver";
-  version = "3.3.0";
+  version = "3.3.1";
 
   src = fetchFromGitHub {
     owner = "mholt";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1yr2jhidqvbwh1y08lpqaidwpr5yx3bhvznm5fc9pk64s7z5kq3h";
+    sha256 = "1izr9znw3mbqpg85qkl1pad5hash531h3dpwbji5w2af2i6x4ga3";
   };
 
-  vendorSha256 = "1ikrgl03r9zkn86kxkqi2kf540g3qzzz24i5wvh6g3d5q49nygl9";
+  vendorSha256 = "1rqhra3rfarq8f750zszkrm0jcsxa4sjbfpmcdlj5z000df699zq";
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version} -X main.commit=${src.rev} -X main.date=unknown" ];
```

Comment:
Is desirable to not update the date here?

Reply:
In this specific case it doesn't really matter, because [`date` would be an empty string](https://github.com/mholt/archiver/blob/v3.3.1/cmd/arc/main.go#L30); but in general we should avoid recording the current date for reproducible builds https://reproducible-builds.org/docs/timestamps/

Reply:
Fine.